tests/MyGame/Example2/Monster.go

```diff
@@ -29,13 +38,29 @@ type Monster struct {
 	_tab flatbuffers.Table
 }
 
+// GetRootAsMonster shortcut to access root table
 func GetRootAsMonster(buf []byte, offset flatbuffers.UOffsetT) *Monster {
 	n := flatbuffers.GetUOffsetT(buf[offset:])
 	x := &Monster{}
 	x.Init(buf, n+offset)
 	return x
 }
 
+// GetTableVectorAsMonster shortcut to access table in vector of  unions
+func GetTableVectorAsMonster(table *flatbuffers.Table) *Monster {
```

Comment:
This seems like a very unsafe function as it works on any table. Can this be done differently?

Reply:
Look at what the other languages do for inspiration

Reply:
I would reconsider, and find a similar, suitable implementation like other languages.